tdrs-backend/tdpservice/parsers/schema_defs/tanf/t4.py

```diff
@@ -20,27 +20,30 @@
               Field(item="0", name='RecordType', type='string', startIndex=0, endIndex=2,
                     required=True, validators=[]),
               Field(item="4", name='RPT_MONTH_YEAR', type='number', startIndex=2, endIndex=8,
-                    required=True, validators=[]),
+                    required=True, validators=[validators.dateYearIsLargerThan(1998),
+                                               validators.dateMonthIsValid()]),
               Field(item="6", name='CASE_NUMBER', type='string', startIndex=8, endIndex=19,
-                    required=True, validators=[]),
+                    required=True, validators=[validators.isAlphaNumeric(),
+                                               validators.notEmpty()]),
               Field(item="2", name='COUNTY_FIPS_CODE', type='string', startIndex=19, endIndex=22,
```

Comment:
@elipe17 im curious why we use a different validator here than we use in section 1 (see below):

```
 Field(item="2", name='COUNTY_FIPS_CODE', type='string', startIndex=19, endIndex=22,
                        required=True, validators=[
                              validators.isNumber(),
                        ]),
```

Reply:
I probably copied it from section 1 prior to it being changed in section 1. Will update to be consistent.